R/LSgameofchance.R

```diff
@@ -191,10 +188,9 @@ LSgameofchance   <- function(jaspResults, dataset, options, state = NULL){
   
   # column specification
   CIPlot0 <- ggplot2::ggplot(data= NULL) + 
-    ggtitle("Probability of Player 1 Winning") +
```

Comment:
we don't need the title here since it's on the container itself (I think all of the JASP plots are titles)